airflow-core/src/airflow/utils/db_cleanup.py

```diff
@@ -36,10 +36,10 @@
 from sqlalchemy.orm import aliased
 from sqlalchemy.sql.expression import ClauseElement, Executable, tuple_
 
+from airflow._shared.timezones import timezone
 from airflow.cli.simple_table import AirflowConsole
 from airflow.configuration import conf
 from airflow.exceptions import AirflowException
-from airflow.utils import timezone
```

Comment:
We should nuke `airflow-core/src/airflow/utils/timezone.py` too since it is no longer used I assume, and add a back-compat entry in `airflow-core/src/airflow/utils/__init__.py`

Reply:
Oh yes, I forgot to do that in this reworked PR. On it.